sci-libs/vtk/vtk-7.0.0.ebuild

```diff
@@ -0,0 +1,362 @@
+SLOT="0"
+IUSE="
+	all-modules aqua boost doc examples imaging ffmpeg gdal java json kaapi mpi
+	mysql odbc offscreen postgres python qt5 rendering tbb test theora tk tcl
+	video_cards_nvidia views web xdmf2 R +X"
+
+REQUIRED_USE="
+	all-modules? ( python xdmf2 boost )
+	java? ( qt5 )
+	python? ( ${PYTHON_REQUIRED_USE} )
+	tcl? ( rendering )
+	test? ( python )
+	tk? ( tcl )
+	web? ( python )
+	^^ ( X aqua offscreen )
+	"
+
+RDEPEND="
+	dev-libs/expat
+	dev-libs/jsoncpp
+	dev-libs/libxml2:2
+	>=media-libs/freetype-2.5.4
+	media-libs/libpng:0=
+	media-libs/mesa
+	media-libs/libtheora
+	media-libs/tiff:0
+	sci-libs/exodusii
+	sci-libs/hdf5:=
+	sci-libs/netcdf-cxx:0=
+	sys-libs/zlib
+	virtual/jpeg:0
+	virtual/opengl
+	>=x11-libs/gl2ps-1.3.8
+	x11-libs/libX11
+	x11-libs/libXmu
+	x11-libs/libXt
+	boost? ( dev-libs/boost:=[mpi?] )
+	examples? (
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		sci-libs/vtkdata
+	)
+	ffmpeg? ( virtual/ffmpeg )
+	gdal? ( sci-libs/gdal )
+	java? ( >=virtual/jre-1.5:* )
+	kaapi? ( <sci-libs/xkaapi-3 )
+	mpi? (
+		virtual/mpi[cxx,romio]
+		python? ( dev-python/mpi4py[${PYTHON_USEDEP}] )
+	)
+	mysql? ( virtual/mysql )
+	odbc? ( dev-db/unixODBC )
+	offscreen? ( media-libs/mesa[osmesa] )
+	postgres? ( dev-db/postgresql:= )
+	python? (
+		${PYTHON_DEPS}
+		dev-python/sip[${PYTHON_USEDEP}]
+	)
+	qt5? (
+		dev-qt/designer:5
+		dev-qt/qtcore:5
+		dev-qt/qtgui:5
+		dev-qt/qtopengl:5
+		dev-qt/qtsql:5
+		dev-qt/qtwebkit:5
+		python? ( dev-python/PyQt5[${PYTHON_USEDEP}] )
+	)
+	tbb? ( dev-cpp/tbb )
+	tcl? ( dev-lang/tcl:0= )
+	tk? ( dev-lang/tk:0= )
+	video_cards_nvidia? ( || ( x11-drivers/nvidia-drivers[tools,static-libs] media-video/nvidia-settings ) )
+	web? (
+		${WEBAPP_DEPEND}
+		python? (
+			dev-python/autobahn[${PYTHON_USEDEP}]
+			dev-python/twisted-core[${PYTHON_USEDEP}]
+			dev-python/zope-interface[${PYTHON_USEDEP}]
+			)
+		)
+	xdmf2? ( sci-libs/xdmf2 )
+	R? ( dev-lang/R )"
+DEPEND="${RDEPEND}
+	doc? ( app-doc/doxygen )
+	java? ( >=virtual/jdk-1.5 )"
+
+S="${WORKDIR}"/VTK-${PV}
+
+PATCHES=(
+	"${FILESDIR}"/${P}-glext.patch
+	"${FILESDIR}"/${PN}-6.1.0-memset.patch
+	"${FILESDIR}"/${PN}-6.1.0-gdal2.patch
+	)
+
+RESTRICT=test
+
+pkg_setup() {
+	use java && java-pkg-opt-2_pkg_setup
+	use python && python-single-r1_pkg_setup
+	use web && webapp_pkg_setup
+}
+
+src_prepare() {
+	sed \
+		-e 's:libproj4:libproj:g' \
+		-e 's:lib_proj.h:lib_abi.h:g' \
+		-i CMake/FindLIBPROJ4.cmake || die
+
+	local x
+	# missing: VPIC alglib exodusII freerange ftgl libproj4 mrmpi sqlite utf8 verdict xmdf2 xmdf3
+	for x in expat freetype gl2ps hdf5 jpeg jsoncpp libxml2 netcdf oggtheora png tiff zlib; do
+		ebegin "Dropping bundled ${x}"
+		rm -r ThirdParty/${x}/vtk${x} || die
+		eend $?
+	done
+	rm -r \
+		ThirdParty/AutobahnPython/autobahn \
+		ThirdParty/Twisted/twisted \
+		ThirdParty/ZopeInterface/zope \
+		|| die
+
+	cmake-utils_src_prepare
+}
+
+src_configure() {
+	# general configuration
+	local mycmakeargs=(
+		-Wno-dev
+#		-DVTK_DIR="${S}"
```

Comment:
Why is this commented out? Please either leave an additional comment for others to know the reasoning, or remove it altogether if it's not necessary anymore.